src/main/java/org/syncninja/service/CommitTreeService.java

```diff
@@ -77,7 +80,7 @@ private void addNodesInPath(String[] pathComponents, String mainDirectoryPath, C
                 LinesContainer linesContainer = CompareFileUtil.compareFiles(path, statusFileDTO);
                 // new File
                 if(commitNode == null) {
-                    commitNode = new CommitFile(path, linesContainer.getLineNumbers(), linesContainer.getNewLines(), linesContainer.getOldLines());
+                    commitNode = new CommitFile(path, statusFileDTO.getFileStatus(),linesContainer.getLineNumbers(), linesContainer.getNewLines(), linesContainer.getOldLines());
```

Comment:
please chagne `statusFileDTO.getFileStatus(),linesContainer.getLineNumbers()` to `statusFileDTO.getFileStatus(), linesContainer.getLineNumbers()`